README.md

```diff
@@ -107,27 +107,30 @@ You can check out [the API documentation](https://docs.meilisearch.com/reference
 
 Go checkout [examples](./examples)!
 
+In this section, the examples contain the [`await` keyword](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await). This operator is used to wait for a `Promise`.<br>
+Despite it is not detailled in the examples above, it can only be used inside an `async` function.
```

Comment:
I wanted to explain because my first version of the getting_started did not show an example with `async/await`. So, if a beginner user copied/pasted this little line of code in the getting started he/she just found out before, it wouldn't work.

With the new getting started example, I agree it's useless.